element-hq/element-x-android
# Improve session recovery screens

## Type of change


## Content

- Update the UI and texts for the 'enter recovery key' screen.
- Add new 'create new recovery key' info screen that can be displayed when there are no other active sessions for the current account.

## Motivation and context

Closes #2579, also takes care of https://github.com/element-hq/element-internal/issues/583.

## Screenshots / GIFs

In the PR.

## Tests



- Log into an account with no other active sessions, or modify `VerifySelfSessionPresenter` so `isLastDevice` is true.
- Check the 'create new recovery key' button appears after logging in.
- Verify the new 'create new recovery key' screen is displayed properly.

## Tested devices

- OS version(s): 14

## Checklist
